packages/application-extension/src/index.tsx

```diff
@@ -412,6 +447,7 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
             commands.execute(CommandIDs.toggleHeader).catch(reason => {
               console.error('Failed to display title header.', reason);
             });
+            ariaLiveRegion!.append(trans.__('Display top header'));
```

Comment:
Again, I think it should be in `.then()`, and the `console.error` could be replaced by a method that writes to both console and the aria live region. Similarly in other places.